src/cli/java/org/commcare/util/cli/ApplicationHost.java

```diff
@@ -323,14 +330,24 @@ private void printErrorAndContinue(String error, Exception e) {
 
     private Screen getNextScreen() {
         String next = mSession.getNeededData(mSession.getEvaluationContext());
-
         if (next == null) {
             //XFORM TIME!
             return null;
         } else if (next.equals(SessionFrame.STATE_COMMAND_ID)) {
             return new MenuScreen();
         } else if (next.equals(SessionFrame.STATE_DATUM_VAL)) {
             return new EntityScreen();
+        } else if (next.equals(SessionFrame.STATE_QUERY_REQUEST)) {
+            return new QueryScreen(qualifiedUsername, password, System.out);
+        } else if (next.equals(SessionFrame.STATE_SYNC_REQUEST)) {
+            new SyncScreen(qualifiedUsername, password);

```

Comment:
So the sync screen on the CLI has no user facing output? feels like it would be worth having it display something to keep it in parity with what webapps does, even if its synchronous.

IE: SyncScreen could show details about the connection, the # of results returned, any error that came up, etc.

currently I think it's the only screen that does what this does, which is process something raw, so I'd want to change what we call it if we don't